recipes/uvwasi/all/patches/0.0.18-0001-fix-cmake.patch

```diff
@@ -0,0 +1,100 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f71c36d..a92e629 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -14,13 +14,13 @@ cmake_dependent_option(UVWASI_BUILD_TESTS
+ 
+ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/out)
+ 
+-if(CMAKE_C_COMPILER_ID MATCHES "AppleClang|Clang|GNU")
++if(0)
```

Comment:
@toge, thanks for your contribution.

What is the motivation behind this change? The patch mentions https://github.com/nodejs/uvwasi/pull/210 but I cannot see this change there.

Reply:
Sorry for the wrong information.
uswasi#210 only adds the installer.

I split the patch file in two.